mpmath/ctx_mp_python.py

```diff
@@ -117,9 +117,9 @@ def mpf_convert_lhs(cls, x):
         return x
 
     man_exp = property(lambda self: to_man_exp(self._mpf_))
-    man = property(lambda self: self._mpf_[1])
-    exp = property(lambda self: self._mpf_[2])
-    bc = property(lambda self: self._mpf_[3])
+    man = property(lambda self: self.man_exp[0])
+    exp = property(lambda self: self.man_exp[1])
+    bc = property(lambda self: self.man.bit_length())
```

Comment:
OT: it looks like the 4-th item in the `_mpf_` tuple could be eliminated. That is, can we assume that
```python
assert _mpf_[1].bit_length() == _mpf_[3]
```
holds and what will be the performance implications?

Reply:
> OT: it looks like the 4-th item in the _mpf_ tuple could be eliminated.

This is true (and maybe the *first* item as well, if we decide to live without signed zero).  But we could do this only via series of backward compatibility breaks.  See https://github.com/mpmath/mpmath/issues/720